src/main/java/com/amazon/jenkins/ec2fleet/EC2FleetCloud.java

```diff
@@ -666,7 +682,7 @@ public void run() {
 
     /**
      * Schedules Jenkins Node and EC2 instance for termination.
-     * If <code>force</code> is false, check if target capacity more than <code>minSize</code> otherwise reject termination.
+     * If <code>force</code> is false, check if target capacity more than <code>minSize</code> or less than <code>minSpareSize</code> otherwise reject termination.
```

Comment:
nit: can be re-worded for clarity 

`* If <code>force</code> is false and target capacity will fall below <code>minSize</code> OR <code>minSpareSize</code> thresholds, then reject termination.`

Reply:
done